src/adapters/error.ts

```diff
@@ -83,6 +83,13 @@ export const metamaskUnknownChainError = StrictSchema<MetamaskUnknownChainError>
   })
 );
 
+export enum ProviderError {
+  Ethereum = "Ethereum",
+  PolygonZkEVM = "PolygonZkEVM",
```

Comment:
If we're rendering this, `PolygonZkEVM` should have a space.
If we're not rendering this, I'd use here the keys `ethereum` and `polygon-zkevm`.

I prefer the second, but do as you prefer.